Stanford/JumpstartEngineering/Install/Layouts/Sitewide.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Configure the sitewide layout.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class Sitewide extends \AbstractInstallTask {
+
+  /**
+   * Configure the sitewide layout.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+
+    $context_status = variable_get('context_status', array());
```

Comment:
@sherakama's question is why does JSE have to be set to False? wouldn't you have already done that in the feature itself?